### PR TITLE
Chore/daef 105 add missing translations for language selection

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR adds missing translations for language selection options.

### CODE REVIEW

**Comment by DominikGuzei** [comment]
Reviewing this now

File: app/stores/AppStore.js [hunk 1]
```diff
@@ -1,17 +1,51 @@
...
+    id: 'profile.languageSelect.form.options.german',
+    defaultMessage: '!!!German',
+    description: 'Label for the language select option "German".',
+  },
+  languageOptionCroatian: {
+    id: 'profile.languageSelect.form.options.croatian',
+    defaultMessage: '!!!Croatian',
+    description: 'Label for the language select option "Croatian".',
+  },
+});
```
Comment (DominikGuzei) [inline app/stores/AppStore.js]: Please move these generic translation messages of languages into `app/i18n/global-messages.js` (where we store reusable messages) and change all the `id`s to something more general like `global.language.english`.

File: app/i18n/locales/hr-HR.json [hunk 2]
```diff
@@ -18,12 +18,12 @@
...
-  "profile.languageSelect.form.options.english": "!!!English",
-  "profile.languageSelect.form.options.german": "!!!German",
-  "profile.languageSelect.form.options.japanese": "!!!Japanese",
-  "profile.languageSelect.form.options.korean": "!!!Korean",
+  "profile.languageSelect.form.options.chinese": "Kineski",
+  "profile.languageSelect.form.options.croatian": "Hrvatski",
+  "profile.languageSelect.form.options.english": "Engleski",
+  "profile.languageSelect.form.options.german": "Njemački",
+  "profile.languageSelect.form.options.japanese": "Japanski",
+  "profile.languageSelect.form.options.korean": "Korejski",
```
Comment (DominikGuzei) [inline app/i18n/locales/hr-HR.json]: Darko said: In this case the languages should be lowercase letter in Croatian (he thinks because it's an adjective) 😂

Comment (nikolaglumac) [inline app/i18n/locales/hr-HR.json]: @darko-mijic is right. I feel ashamed :(

File: app/stores/AppStore.js (hunk 1 above)
Comment (nikolaglumac) [inline app/stores/AppStore.js]: Thanks for the tip! Global messages are much much better place to put these translations ;)

**Comment by DominikGuzei** [comment]
@nikolaglumac Github says that there are some conflicts that have to be resolved

**Comment by nikolaglumac** [comment]
@DominikGuzei conflicts are resolved and updated is pushed to GH.

**Comment by DominikGuzei** [comment]
It seems like appveyor/pr has some problems because of the previous merge conflicts 😢

**Pull request merged**